backend/service/k8s/meta.go

```diff
@@ -15,6 +15,11 @@ func ApplyListOptions(listOpts *k8sapiv1.ListOptions) metav1.ListOptions {
 	opts := metav1.ListOptions{}
 	if len(listOpts.Labels) > 0 {
 		opts.LabelSelector = labels.FormatLabels(listOpts.Labels)
+	} else if len(listOpts.SelectorString) > 0 {
```

Comment:
comment on behavior, as labels takes presidents or even error handle that both fields cant be set simultaneously, unless it's valid to merge them together.

Reply:
I had a new idea: how about we just concat them, as in the selector string will be a supplement rather than a backup replacement

Reply:
since set-based selectors are not supported by all resources and since `listOptions` is currently used by all List calls how do we plan on restricting the usage?

Reply:
I guess the plan is to test each one, and since the string is supplemental it won't change anything for the current uses.  Also, with a validation check for the errors that will help stop problems.

I couldn't find a listing of resources that support it vs those that don't other than those already listed in the docs.